Dockerfile.asserts

```diff
@@ -1,26 +1,29 @@
 # Build the manager binary
 FROM golang:1.16 as builder
 
-WORKDIR /go/src/github.com/jaegertracing/jaeger-operator
+WORKDIR /workspace
 
 # Download the dependencies. Doing this, if there are changes in the source
 # code but not in the dependencies to download, the tool to build the image will
 # use the cached dependencies
-COPY go.mod /go/src/github.com/jaegertracing/jaeger-operator/go.mod
-COPY go.sum /go/src/github.com/jaegertracing/jaeger-operator/go.sum
+COPY go.mod go.mod
+COPY go.sum go.sum
 RUN go mod download
 
-COPY . /go/src/github.com/jaegertracing/jaeger-operator/
+COPY tests tests
+
+ENV GO111MODULE=on
```

Comment:
I think thats a default since 1.16

Reply:
Sorry, I merged something should not be here. Thanks!